ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/420__model_serving.robot

```diff
@@ -11,7 +11,7 @@ Resource          ../../../Resources/OCP.resource
 Resource          ../../../Resources/CLI/ModelServing/modelmesh.resource
 Suite Setup       Model Serving Suite Setup
 Suite Teardown    Model Serving Suite Teardown
-Test Tags         ModelMesh
+Test Tags         ModelMesh    Dashboard
```

Comment:
@FedeAlonso I would avoid having multiple tags on the suite, since we try to make it clear which component are the tests belong to.
If we cannot move the tests to a new suite, you can add specific tag per tests instead of on the suite.

Reply:
Sorry @manosnoam I misunderstood the task then. 
Fixed in  [9783baf](https://github.com/red-hat-data-services/ods-ci/pull/1529/commits/9783baf6a46d0e3310aac645412b20473c24d886)